conf/modules/sd_logger_spi_direct.xml

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="loggers">
```

Comment:
Should be something like  `name="sd_logger_spi_direct" dir="loggers"`.
Also, other logger module files are named `logger_<something>.xml`, so it should probably be `logger_sd_spi_direct.xml` here.